packages/select/src/select.vue

```diff
@@ -119,6 +119,7 @@
       @after-leave="doDestroy">
       <el-select-menu
         ref="popper"
+        @placement="handlePlacement"
```

Comment:
Maybe we should mimic the `visible-change` event and call this `placement-change`?

Reply:
Makes sense to me.